lib/Doctrine/Export/Mssql.php

```diff
@@ -46,11 +46,11 @@ public function createDatabase($name)
         $query = "CREATE DATABASE $name";
         $options = $this->conn->getOptions();
         if (isset($options['database_device']) && $options['database_device']) {
-            $query.= ' ON '.$this->conn->options['database_device'];
-            $query.= $this->conn->options['database_size'] ? '=' .
-                     $this->conn->options['database_size'] : '';
+            $query.= ' ON '.$this->conn->getOption('database_device');
+            $query.= $this->conn->getOption('database_size') ? '=' .
+                     $this->conn->getOption('database_size') : '';
         }
-        return $this->conn->standaloneQuery($query, array(), true);
+        return $this->conn->standaloneQuery($query, array());
```

Comment:
No third parameter to `Doctrine_Connection::standaloneQuery` (only place that method exists)